web/src/components/MapView.tsx

```diff
@@ -88,7 +111,7 @@ const MapView = React.forwardRef((props: MapViewProps, ref: React.Ref<MapRef>):
       const feature = event.features && event.features[0]
       if (feature) {
         selectFeature(feature)
-        changeSnapPoint(1)
+        changeSnapPoint(2)
```

Comment:
Why did you change to snap point 2? Now the user cannot see the selected feature on the map anymore, as it is covered by the `bottomActionSheet`.

Reply:
This was requested by me. I think this makes sense since this only happens if you select a poi on the map. And if you do that, you already saw where the poi is on the map. Nothing was changed for selecting a poi in the list/opening it via query param.